app/src/components/adminConsole/publishedCups/PublishedCupsList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Cup from "../../../types/Cup";
+import { Entity } from "../../../types/Entity";
+import styled from "styled-components";
+import { MessageDisplay } from "../AdminConsoleStyles";
+
+function formatTimePublished(time: {
+    seconds: number;
+    nanoseconds: number;
+}): string {
+    const d = new Date(time.seconds * 1000);
+    return (
+        "Published: " +
+        d.toLocaleString(undefined, {
+            year: "numeric",
+            month: "long",
+            day: "2-digit",
+        })
+    );
+}
+
+const CupCard = styled.div`
+    box-shadow: 2px 2px 8px rgba(0, 0, 0, 0.25);
+    border-radius: 10px;
+    postion: relative;
+    height: 100%;
+    background: #7040d6;
+`;
+
+const CupText = styled.div`
+    font-family: var(--open-sans);
+    font-style: normal;
+    font-weight: bold;
+    line-height: 27px;
+    color: #faf8f8;
+    padding: 20px 33px;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    align-items: left;
+`;
+
+const CupCardName = styled.p`
+    color: var(--white);
+    padding: 0;
+    margin: 0 0 13px 0;
+    font-size: 18px;
+`;
+
+const CupCardDate = styled.p`
+    color: var(--white);
+    padding: 0
+    margin: 0 0 13px 0;
+    font-size: 14px;
+`;
```

Comment:
Most of these styles are the same. Could you commonise that?

Reply:
As it turns out, the common stuff was unnecessary anyway